index.html

```diff
@@ -4,35 +4,97 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>MyStayl</title>
   <link rel="stylesheet" href="my.css">
+  <script src="app.js"></script>
+  <title>document</title>
 </head>
- <style>
-Ul {
-  list-style-type: none;
-  margin: 0;
-  padding: 0;
-} 
-</style>
-</head>
-<body> 
-  <div class="qqqq">
-    <a class="wwwww" href="default.asp">Home</a>
-    <a href="news.asp">News</a>
-    <a href="contact.asp">Contact</a>
-    <a href="about.asp">About</a>
-  </div>
-  <div class="eeee">
-    <a class="rrrr" href="default.asp">Home</a>
-    <a href="news.asp">News</a>
-    <a href="contact.asp">Contact</a>
-    <a class="tttt" href="about.asp">About</a>
+<body>
+    <header class="header">
+      <div class='container'>
+        <div class="header__body">
+          <a href="#" class="header__logo">
+            <img src="logo.jpg" alt="LogoOnHeader">
+          </a>
+          <div class="header__burger" onclick="switchClass()">
+            <div class="header__burger__middle"></div>
+          </div>
+            <nav class="header_menu_size_m">
+              <ul class="header__list">
+                <li>
+                  <a href=#Главная class="header__link">Главная</a>
+                </li>
+                <li>
+                  <a href="#О нас" class="header__link">О нас</a>
+                </li>
+                <li>
+                  <a href="#Япоские ведра" class="header__link">Японские ведра</a>
+                  <ul class="header__submenu">
+                   <li> <a href="#">тойота</a></li>
+                   <li> <a href="#">субару</a></li>
+                   <li> <a href="#">хонда</a></li>
+                  </ul>
+                </li>
+                <li>
+                  <a href="#Немецкие ведра" class="header__link">Немецкие ведра</a>
+                  <ul class="header__submenu__first">
+                    <li> <a href="#">Ауди</a></li>
+                    <li> <a href="#">Бемельве</a></li>
+                    <li> <a href="#">Мередес</a></li>
+                   </ul>
+                </li>
+                <li>
+                  <a href="#Американские ведра" class="header__link">Американские ведра</a>
+                  <ul class="header__submenu__second">
+                    <li> <a href="#">Шевроле</a></li>
+                    <li> <a href="#">Форд</a></li>
+                    <li> <a href="#">додж</a></li>
+                   </ul>
+                </li>
+              </ul>
+            </nav>
+        </div>
+      </div>
+    <nav class="header_menu_size_s" id="switch">
+    <label for="#"> Главная
+    </label>
+    <label for="#"> О нас

```

Comment:
The For attribute couldn't be #. We use the For attributes in order to create connection between a label and an input field. Please read more about semantic of html's form.